l10n_eu_oss/wizard/l10n_eu_oss_wizard.py

```diff
@@ -108,7 +111,7 @@ def _prepare_repartition_line_vals(self, original_rep_lines):
             for line in original_rep_lines
         ]
 
-    def _prepare_tax_vals(self, country_id, tax_id, rate):
+    def _prepare_tax_vals(self, country_id, tax_id, rate, tax_group_id):
```

Comment:
Please don't use the suffix `_id` when you are passing a recordset.

```suggestion
    def _prepare_tax_vals(self, country_id, tax_id, rate, tax_group):
```